modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -47,6 +47,15 @@ This resulted in an inconsistent end-user experience.
 
 {productname} now presents end-users with uniform merge tag highlighting across all supported browsers.
 
+=== Insert merge tag button and menu item was not disabled when selection was inside a non-editable context.
+//#TINY-9893
+
+In previous versions of {productname}, the insert merge tag button and menu item remained active when the user's selection was inside a `contenteditable="false"` context.
+
+As a consequence the mergetags button and menu item would appear enabled, but clicking on them had no effect.
+
+A fix was implemented in {productname} 6.5 that now checks if the users content selection is within a `contenteditable="false"` context. If it is, the button and menu item will now be disabled.
```

Comment:
```suggestion
In previous versions of {productname}, the *Insert → Merge tag* menu item, and equivalent toolbar button remained active when the user’s selection or insertion point was inside an element with a `contenteditable="false"` attribute set.

As a consequence the *Merge tags* toolbar button and menu item appeared enabled, but clicking on them had no effect.

A fix was implemented in {productname} 6.5. {productname} now checks if the user’s current selection is within an element with a `contenteditable="false"` attribute set. If it is, the *Merge tags* toolbar button and menu item are disabled, as expected.
```

1. spelling out _element with a `contenteditable="false"` attribute_ in full for correctness’s sake.
2. copy edits to UX and UI item spelling and presentation to exactly match presentation in the editor.
3. Other suggested edits.